scripts/scil_endpoints_map.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import argparse
+import logging
+import json
+import os
+
+import nibabel as nib
+import numpy as np
+
+from scilpy.io.streamlines import load_tractogram_with_reference
+from scilpy.io.utils import (add_json_args,
+                             add_overwrite_arg,
+                             add_reference_arg,
+                             assert_inputs_exist,
+                             assert_outputs_exist)
+
+DESCRIPTION = '''
+Computes the endpoint map of a bundle. The endpoint map
+is simply a count of the number of streamlines that
+start or end in each voxel. The idea is to estimate the
+cortical areas affected by the bundle (assuming
+streamlines start/end in the cortex)
+'''
```

Comment:
I know the concept of head and tail was confusing to philippe, maybe saying explicitely that if the streamlines are not aligned in X, Y or Z the head/tail are basically random and not really two coherent group.

Reply:
Copy paste :smile: